### PR TITLE
Add Button Variants

### PR DESCRIPTION
**Pull request by dovbush-olha**
Task: https://olenitut.atlassian.net/jira/software/projects/LIN/boards/3?selectedIssue=LIN-6

### CODE REVIEW

**Review by olenitut — CHANGES_REQUESTED** [review]
Overall, great job! 
I loved what you did with typing and the button system you created!
As for microchanges, I would prefer us to avoid inline styles, lets discuss how we can do it in a best practice way) 
also, we will need to go through prettier config and import code styling 

Thanks <3

File: package.json [hunk 1]
```diff
@@ -22,6 +23,7 @@
     "eslint": "^8",
     "eslint-config-next": "14.0.0",
     "postcss": "^8",
+    "prettier": "^3.0.3",
```
Comment (olenitut) [inline package.json:26]: btw, prettier is usually added as a dev dependency, we dont need it to appear in the final build https://prettier.io/docs/en/install.html

File: .prettierrc
[the remark this answers is not in the document]
Comment (dovbush-olha) [reply .prettierrc:1]: What settings are you used to?

File: src/components/ui/button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+type Props = {
+  variant?: ButtonVariant;
+  size?: ButtonSize;
+  children?: string | React.ReactNode;
+  onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  id?: string;
+  className?: string;
+  style?: React.CSSProperties;
+};
```
[the remark this answers is not in the document]
Comment (dovbush-olha) [reply src/components/ui/button/Button.tsx:13]: Thank you

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,16 +1,82 @@
...
+        <Button size={ButtonSize.Medium} variant={ButtonVariant.Circle} id='control' />
+        <Button size={ButtonSize.Medium} variant={ButtonVariant.Circle} id='close' />
+        <Button size={ButtonSize.Medium} variant={ButtonVariant.Circle} id='plus' />
+        <Button size={ButtonSize.Medium} variant={ButtonVariant.Circle} id='smile' />
+        <Button size={ButtonSize.Medium} variant={ButtonVariant.Circle} id='star' />
+        <Button
+          size={ButtonSize.Medium}
+          variant={ButtonVariant.Circle}
+          id='controlMini'
+          style={{ width: '40px', height: '40px' }}
```
[the remark this answers is not in the document]
Comment (dovbush-olha) [reply src/app/page.tsx:59]: Okay

**Review by olenitut — APPROVED** [review]

**Pull request merged**